website/addons/base/serializer.py

```diff
@@ -56,6 +56,9 @@ def serialized_user_settings(self):
 
 
 class OAuthAddonSerializer(AddonSerializer):
```

Comment:
Please implement a subclass of OAuthAddonSerializer (refer to CitationsAddonSerializer as an example) rather than modifying the OAuthAddonSerializer class.